Makefile

```diff
@@ -8,4 +8,7 @@ build:
 
 install: build
 	mkdir -p ~/.tflint.d/plugins
-	mv ./tflint-ruleset-template ~/.tflint.d/plugins
+	mv ./tflint-ruleset-kafka-config ~/.tflint.d/plugins
```

Comment:
Should this directory be in `.gitignore`?

Reply:
yep, good spot https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/4/commits/e4b1d905d218d98e04f056d037d935e9f065e1ca